index.html

```diff
@@ -520,9 +520,114 @@
 				grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
 			}
 
+			#minimizeButton {
+				background: transparent; 
+				border: 0px;
+				width: 30px;
+				height: 30px;
+				fill: var(--item-col)
+			}
+
+			#minimizeButton:hover {
+				background: var(--item-col);
+				fill: var(--highlight-col)

```

Comment:
Add transition like with the other buttons. same goes for the other :hover's you added